src/DssSpell.t.sol

```diff
@@ -139,6 +121,28 @@ contract DssSpellTest is DSTest, DSMath {
     ClipperMomAbstract   clipMom = ClipperMomAbstract( addr.addr("CLIPPER_MOM"));
     DssAutoLineAbstract autoLine = DssAutoLineAbstract(addr.addr("MCD_IAM_AUTO_LINE"));
 
+    // Specific for this spell
+
+    address constant GRO_MULTISIG        = 0x7800C137A645c07132886539217ce192b9F0528e;
+    address constant MKT_MULTISIG        = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
+    address constant GOV_MULTISIG        = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+    address constant RWF_MULTISIG        = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
+    address constant RISK_MULTISIG       = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+    address constant PE_MULTISIG         = 0xe2c16c308b843eD02B09156388Cb240cEd58C01c;
+    address constant ORA_MULTISIG        = 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf;
+    address constant ORA_ER_MULTISIG     = 0x53CCAA8E3beF14254041500aCC3f1D4edb5B6D24;
+
+    uint256 constant amountGro    = 126117;
+    uint256 constant amountMkt    = 44375;
+    uint256 constant amountGov    = 273334;
+    uint256 constant amountRwf    = 155000;
+    uint256 constant amountRisk   = 182000;
+    uint256 constant amountPe     = 510000;
+    uint256 constant amountOra    = 419677;
+    uint256 constant amountOraEr  = 800000;
+    uint256 constant amountTotal  = 2510503;
```

Comment:
Why not `amountTotal = amountGro + amountMkt...` ?

Reply:
Wanted to make sure my total amount here matched the comment in the spell